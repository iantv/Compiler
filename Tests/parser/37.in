void main(){
	while (1)
		continue;
}