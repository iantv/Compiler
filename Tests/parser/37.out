main: function
	{
		while: loop
			continued cond:
				1
			{
				continue
			}
	}
returns
	void
